testutils/infra/container_manager/docker_compose_base_manager.py

```diff
@@ -87,8 +90,12 @@ def get_logs_of_service(self, service):
 
     def get_virtual_network_host_ip(self):
         """Returns the IP of the host running the Docker containers"""
-        temp = "docker ps -q " "--filter label=com.docker.compose.project={project} "
-        cmd = temp.format(project=self.name)
+        temp = (
+            "docker ps -q "
+            "--filter label=com.docker.compose.project={project} "
+            "--filter label=com.docker.compose.service={service}"
+        )
+        cmd = temp.format(project=self.name, service="mender-api-gateway")
```

Comment:
the service here serves as a way to get only one result, right? or we now need to explicitly get it from mender-api-gateway? (the `head -1` below is just a double-safety then?)

Reply:
I guess before it didn't matter but now the old expression might return the mender-gateway IP address.

Reply:
Yep, without it there is the risk of picking up the host ID in the wrong network (the internal one).
As all docker composition will have at least the mender-api-gateway, and it is always connected to the main "mender" network, wee use the service to be sure to pick up the right network.